server/handler/status.go

```diff
@@ -114,12 +115,12 @@ func (h *Status) processOthers(ctx context.Context, event github.StatusEvent) er
 
 	// In practice, there should only be one or two open PRs for a given commit. In exceptional cases, if there are
 	// more than 100 open PRs, only process the most recent 100.
-	prs, _, err := client.PullRequests.List(
+	prs, _, err := client.PullRequests.ListPullRequestsWithCommit(
```

Comment:
Nice, I didn't know this endpoint existed in the v3 API. Per the [docs](https://developer.github.com/v3/repos/commits/#list-pull-requests-associated-with-commit), it looks like it returns both closed and open PRs:

> Lists all pull requests containing the provided commit SHA, which can be from any point in the commit history. The results will include open and closed pull requests.

I think the 100 PR limit is _probably_ still safe in practice (although the comment is inaccurate now), but we do need to filter the results to only evaluate open PRs in the loop on L132.